gitops/credentials/kubeconfig/README.md

```diff
@@ -0,0 +1,10 @@
+# Clusters credentials 
+
+This directory and sub-directories aim at containing the kubeconfig files used for registering the clusters to Argo CD.
+
+---
+**_NOTES:_**
+
+The information contained in kubeconfig files is confidential. Measures should be taken to protect it from being disclosed. This directory and sub-directories should not contain these files in a public repository. Don't forget to amend the `.gitignore` file if you want to add other files to a private fork of this repository.
```

Comment:
Which is why I talk about encryption earlier. We should steer the user toward the right thing as much as possible.

Reply:
We need to do things step by step